samples/FeatureFlags.AspNetCore/Startup.cs

```diff
@@ -36,13 +60,15 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
                 app.UseDeveloperExceptionPage();
             }
 
-            app.UseFeatureFlags(options);
-            //app.UseFeatureFlagsUI(options);
+            app.UseRimDevFeatureFlags();
+            app.UseRimDevFeatureFlagsUi();
```

Comment:
Do we want to keep the `Ui`-part as-is or change to `UI` to mirror casing with `MapFeatureFlagsUI` (or vice-versa)?